README.adoc

```diff
@@ -116,7 +111,25 @@ NOTE: This file is where we alias the `latest-dev` and `latest` paths in URLs.
 /imdg/latest-dev/*  /imdg/5.1-snapshot/:splat 200!
 ----
 
-The documentation site will now be built, including content from your tagged version.
+- In the `develop` branch of the `hazelcast/hazelcast-docs` repository, submit a pull request to update the `search-config.json` file with the new versions of IMDG.
```

Comment:
So release pipeline should work with `develop` branch for all types of releases (if it's a `latest` release). It never changes or uses anything on `main` branch. Right?

Reply:
Yes. The `develop` branch is the staging site, which is where we can test the changes before going live.

We merge to `develop` first and then run a GitHub Action to merge `develop` into `master`.